app/gui/view/component-browser/component-list-panel/grid/src/entry.rs

```diff
@@ -495,15 +484,15 @@ impl grid_view::Entry for View {
             data.label.set_view_width <+ max_text_width.some();
             content_changed <- data.label.content.constant(());
             style_changed <- style.constant(());
-            highlight_range <= all_with3(
-                &input.set_model,
-                &content_changed,
-                &style_changed,
-                |m, (), ()| m.highlighted.deref().clone()
-            );
+            label_updated <- all3(&input.set_model, &content_changed, &style_changed);
+            highlight_range <= label_updated.map(|(m, (), ())| m.highlighted.deref().clone());
             data.label.set_property <+ highlight_range.map2(&style, |range, s| {
                 (range.into(), Some(text::SdfWeight::new(s.highlight_bold).into()))
             });
+            is_header <- label_updated.filter(|(m, (), ())| m.kind == Kind::Header);
+            data.label.set_property <+ is_header.map2(&style, |_, s| {
+                ((..).into(), Some(text::SdfWeight::new(s.highlight_bold).into()))
```

Comment:
We are now supporting true bold fonts. SdfWeight looks suboptimal in many cases. Until you don't care about keeping the same width of letters, don't use it. We should use it when for example highlighting letters matched by searching for a word - because we don't want the matched letters to be wider. But for headers and other static elements, normal bold fonts should be used.